.github/workflows/taiko-client--docker.yml

```
@@ -2,7 +2,7 @@ name: "Build and Push Multi-Arch Docker Image"
 
 on:
   push:
-    branches: [main]
+    branches: [main, gossip_request_response]
```

```suggestion
    branches: [main]
```
When it is ready to merge, can quickly revert this change